__tests__/app.test.js

```diff
@@ -102,4 +91,47 @@ describe('/api', () => {
     })
 })
 
+describe('/api/articles', () => {
+    test('200: GET /api/articles - responds with an array of article objects containing the relevant keys', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            const {articles} = body;
+            expect(articles.length > 0).toBe(true);
 
+            articles.forEach((article) =>{
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(String),
+                    
+                })
+            })
+        })
+    })
+
+    test('200: GET /api/articles - array returned should be sorted by date in descending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            const {articles} = body;
+            const dates = [];
+            expect(articles.length > 0).toBe(true);
+
+            articles.forEach((article) =>{
+                dates.push(article.created_at);
```

Comment:
not quite sure what this is for but you definitely don't need it

Reply:
I think I was thinking of another way to do the test but being sloppy I left this in, I'll come back to this one and straighten out the problems here